plugins/mimecast/komand_mimecast/util/api.py

```diff
@@ -140,9 +140,9 @@ def _is_last_token(self, request: requests.Response) -> bool:
         """
 
         try:
-            last_token = request.json().get("meta", {}).get(IS_LAST_TOKEN_FIELD, False)
+            last_token = IS_LAST_TOKEN_FIELD in json.dumps(request.json())
```

Comment:
I intially changed this on 5.3.3 to raise 401s correctly, but reverting back to make sure it has no adverse impacts.